site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -286,27 +286,84 @@ When HTTP POST Bindings are used, this is the default verification key used if a
+The unique Id of the Key used to sign the SAML Logout response. 
+
+[field]#application.samlv2Configuration.logout.requireSignedRequests# [type]#[boolean]# [required]#Required# [default]#Defaults to `false`# [since]#Available since 1.25.0#::
+Set this parameter equal to `true` to require the SAML v2 Service Provider to sign the Logout request. When this value is `true` all Logout requests missing a signature will be rejected.
++
+When set to `true`, the parameter [field]#application.samlv2Configuration.logout.defaultVerificationKeyId# is required.
+
+[field]#application.samlv2Configuration.logout.singleLogout.enabled# [type]#[Boolean]# [optional]#Optional# [default]#Defaults to `false`# [since]#Available since 1.25.0#::
+Whether or not SAML Single Logout for this SAML IdP is enabled.
+
+[field]#application.samlv2Configuration.logout.singleLogout.keyId# [type]#[UUID]# [optional]#Optional# [default]#Defaults to the key defined by `application.samlv2Configuration.keyId`# [since]#Available since 1.25.0#::
+The unique Id of the Key used to sign the SAML Single Logout response. 
+
+[field]#application.samlv2Configuration.logout.singleLogout.url# [type]#[String]# [optional]#Optional# [since]#Available since 1.25.0#::
+The URL at which you want to receive the `LogoutRequest` from FusionAuth.
++
+Required if [field]#application.samlv2Configuration.logout.singleLogout.enabled# is `true`.
+
+[field]#application.samlv2Configuration.logout.singleLogout.xmlSignatureC14nMethod# [type]#[String]# [optional]#Optional# [default]#Defaults to `exclusive_with_comments`# [since]#Available since 1.25.0#::
++
+:signed_entity: SAML Single Logout response
++
+include::_xmlSignatureC14nMethod-description.adoc[]
++
+include::../shared/_xmlSignatureC14nMethod-values.adoc[]
++
+:signed_entity!:
+
+[field]#application.samlv2Configuration.logout.xmlSignatureC14nMethod# [type]#[String]# [optional]#Optional# [default]#Defaults to `exclusive_with_comments`# [since]#Available since 1.25.0#::
++
+:signed_entity: SAML Logout response
++
+include::_xmlSignatureC14nMethod-description.adoc[]
++
+include::../shared/_xmlSignatureC14nMethod-values.adoc[]
++
+:signed_entity!:
+
 [field]#application.samlv2Configuration.logoutURL# [type]#[String]# [optional]#Optional# [default]#Defaults to the system logout URL or `/`# [since]#Available since 1.6.0#::
-The URL that the browser is taken to after the user logs out of the SAML service provider. Often service providers need this URL in order to correctly hook up single-logout. **Note** that FusionAuth does not support the SAML single-logout profile because most service providers to not support it properly.
+The URL that the browser is taken to after the user logs out of the SAML service provider. Often service providers need this URL in order to correctly hook up single-logout.
```

Comment:
This is also the URL that will be sent the SAML v2 LogoutResponse using the same bindings that were used to initiate the logout request with the IdP. If POST bindings were used, POST bindings will be used here, and so forth.